app/assets/javascripts/components/story/CollapsedStory/index.js

```diff
@@ -82,8 +83,13 @@ export const Container = ({
 export const CollapsedStory = ({ index, sprintIndex, columnId, ...props }) => {
   const { story } = { ...props }
 
-  const isDragDisabled = useMemo(() =>
-    story.state === 'accepted' || columnId === 'search' || story.loading, [story, columnId]
+  const isDragDisabled = useMemo(
+    () =>
```

Comment:
Let's use the constants from here https://github.com/Codeminer42/cm42-central/blob/master/app/assets/javascripts/libs/beta/constants.js#L8-L17 and here https://github.com/Codeminer42/cm42-central/blob/master/app/assets/javascripts/libs/beta/constants.js#L39-L44 in this conditional. I know you didn't add all of it in your PR but since this file was touched I think it's worth to refactor it now.